README.md

```diff
@@ -0,0 +1,44 @@
+[![Coverage Status](https://coveralls.io/repos/github/Mumbere7/quickCredit/badge.svg?branch=develop)](https://coveralls.io/github/Mumbere7/quickCredit?branch=develop)
+[![Build Status](https://travis-ci.com/Mumbere7/quickCredit.svg?branch=master)](https://travis-ci.com/Mumbere7/quickCredit)
+
+# quickCredit
+
+## Overview
+
+Loan lending web application for andela bootcamp test
+This application should allow user to borrow money from our services at an affordable fee that is competitive Users of the said applocation should be able to repay their loans a given period of time.
+
+## Features of this project
+
+> user
+
+1. User (client) can sign up.
+2. User (client) can login.
+3. User (client) can view loan repayment history, to keep track of his/her liability or
+   responsibilities.
+
+> Admin
+
+1. Admin mark a client as verified.
+2. Admin view all loan applications.
+3. Admin view all current loans (not fully repaid).
+4. Admin view all repaid loans.
+5. Admin view a specific loan application.
+6. Admin approve/reject loan application.
+7. Admin post loan repayment transaction in favour of a client.
+
+> > To view a live demo of this project go to https://mumbere7.github.io/quickCredit/
+> > Give your feed back to the admin in acse of any changes
+
+# <<<<<<< HEAD
```

Comment:
It seems you didn't resolve the merge conflicts well. Please resolve them. You can reach out to Murhula Metre for help about this if you get stuck.

Reply:
I tried but it failed, but I will do as you have requested.